treatment-service/config/config.go

```diff
@@ -109,12 +117,34 @@ func (c *Config) ListenAddress() string {
 	return fmt.Sprintf(":%d", c.Port)
 }
 
+func (c *Config) Validate() error {
+	validate := validator.New()
```

Comment:
The config is already using [viper](https://github.com/spf13/viper), it has capability to do simple validator. Can we utilize that rather than introducing another variant for validating config?

Reply:
Yes, I have removed the custom validation.